bin/commands/stop.js

```diff
@@ -18,14 +19,20 @@ module.exports = function stop(args, rawArgs) {
     // accept the access key from command line if provided
     utils.setAccessKey(bsConfig, args);
 
+    let initDetails = await getInitialDetails(bsConfig, args, rawArgs);
```

Comment:
lets store the result after first API call if data is not available then hit the api. Lets make this code section common under a function.